circle.yml

```diff
@@ -5,6 +5,8 @@ dependencies:
   pre:
     - sudo apt-get update; sudo apt-get -y install python-pip libglew-dev gdb
     - sudo pip install awscli
+  override:
+    - npm update && npm install
```

Comment:
We're not using any of those scripts right now but disabling them feels like a big "gotcha" waiting to happen. Not sure how to weight that cost against the increased build time.